deps/chakrashim/core/lib/Runtime/Base/ThreadContext.h

```diff
@@ -309,13 +309,17 @@ class JITTimer
 class ThreadConfiguration
 {
 public:
-    ThreadConfiguration(bool enableExperimentalFeatures)
+    ThreadConfiguration(bool enableExperimentalFeatures, bool enableSimdjsFeature)
     {
         CopyGlobalFlags();
         if (enableExperimentalFeatures)
         {
             EnableExperimentalFeatures();
         }
+        if (!enableSimdjsFeature)
```

Comment:
(Understand this is temporary) Just wonder if `m_Simdjs = enableSimdjsFeature` here would appear more consistent. This code is in chakracore. Out of node context, it is legal / ok to have `enableExperimentalFeatures == false`. What if people pass JsRuntimeAttributes of this combination? If you design that `JsRuntimeAttributeEnableSimdjsFeature` depends on `JsRuntimeAttributeEnableExperimentalFeatures`, that's ok, maybe document it with the new attribute.